iOS/_Stream Temp/Filter+Extensions.swift

```diff
@@ -13,15 +13,41 @@ extension Filter where Scope: AnyChannelListFilterScope {
 
     /// Filter to match conversations that contain all and only the passed in members.
     static func containOnlyMembers(_ members: [ConversationMember]) -> Filter<Scope> {
+        let userIds = members.map { member in
+            return member.id
+        }
+        return .containsOnly(userIds: userIds)
+    }
+
+    /// Filter to match conversations that contain all and only members with the passed in user ids.
+    static func containsOnly(userIds: [UserId]) -> Filter<Scope> {
         var memberFilters: [Filter<Scope>] = []
-        for userID in members.userIDs {
+        for userID in userIds {
             memberFilters.append(.containMembers(userIds: [userID]))
         }
 
         // Make there aren't other members in this channel who weren't included in the list.
-        let memberCountFilter: Filter<Scope> = .equal(.memberCount, to: members.count)
+        let memberCountFilter: Filter<Scope> = .equal(.memberCount, to: userIds.count)
         memberFilters.append(memberCountFilter)
 
         return .and(memberFilters)
     }
+
+    /// Filter to match conversations that contain all the passed in members. May contain other members.
+    static func containsAtLeastTheseMembers(_ members: [ConversationMember]) -> Filter<Scope> {
+        let userIds = members.map { member in
+            return member.id
+        }
+        return .containsAtLeastThese(userIds: userIds)
+    }
+
+    /// Filter to match conversations that contain all the members with the passed in user ids. May contain other members.
+    static func containsAtLeastThese(userIds: [UserId]) -> Filter<Scope> {
+        var memberFilters: [Filter<Scope>] = []
+        for userID in userIds {
+            memberFilters.append(.containMembers(userIds: [userID]))
+        }
+
+        return .and(memberFilters)
+    }
```

Comment:
Gets conversation that must contain the members, but might also contain other members.